FlingEngine/Graphics/src/FlingImgui.cpp

```diff
@@ -235,7 +235,7 @@ namespace Fling
             Initalizers::DescriptorPoolSize(VK_DESCRIPTOR_TYPE_INPUT_ATTACHMENT, 1000),
         };
 
-        VkDescriptorPoolCreateInfo descriptorPoolInfo = Initalizers::DescriptorPoolCreateInfo(poolSizes, 2);
+        VkDescriptorPoolCreateInfo descriptorPoolInfo = Initalizers::DescriptorPoolCreateInfo(poolSizes, 10);
```

Comment:
just so I understand, did you make this 10 just so we have more in the pool, or is there a more specific reason?